src/common/data.rs

```diff
@@ -68,45 +60,46 @@ impl Display for Data {
     /// Displays some Passerine Data in a pretty manner, as if it were printed to console.
     fn fmt(&self, f: &mut Formatter<'_>) -> Result {
         match self {
-            Data::Heaped(_)   => unreachable!("Can not display heaped data"),
-            Data::NotInit     => unreachable!("found uninitialized data on top of stack"),
-            Data::Real(n)     => write!(f, "{}", n),
-            Data::Integer(n)  => write!(f, "{}", n),
-            Data::Boolean(b)  => write!(f, "{}", if *b { "true" } else { "false" }),
-            Data::String(s)   => write!(f, "{}", s),
-            Data::Lambda(_)   => unreachable!("Can not display naked functions"),
-            Data::Closure(c)  => write!(f, "Function ~ {}", c.id),
-            Data::Kind(_)     => unreachable!("Can not display naked labels"),
```

Comment:
Compilers tend to be sets of transformations on highly-structured tree-like data, which means that Passerine uses `match` extensively. Although I appreciate the consistency tools like `rustfmt` offer for enforcing code structure, they tend not to align match arms. Using `rustfmt` with Passerine is an ongoing discussion https://github.com/vrtbl/passerine/issues/40, but until https://github.com/rust-lang/rustfmt/issues/894 is resolved I value the improved readability of aligned match expressions over programmatically enforced stylistic consistency. Hope this makes sense!